### ISSUE TITLE
Describe the difference between this project and other "open police data projects" in the headline

### ISSUE THREAD

**Issue by jessept** [Police-Data-Accessibility-Project/organizational-issues#64]
We could be losing support because "this data already exists somewhere".  There are 2 value props for the way we are currently tackling this process:
1.) Transparent data lineage. Because our scrapers contain URLs, it is immediately clear where this data comes from, which helps ensure we have legitimate data sources.

2.) High-grain data. Other resources are typically already aggregated, which make it difficult to clean/trust. They also do not lend themselves to more in-depth analyses by joining to other data sources, and restrict the types of analysis to be done because they are aggregated (usually based on news sources rather than public record).

**Comment by nmbeau** [comment]
Yes! This is an important effort for sustainability of the project.
